### ISSUE TITLE
There is a problem with the official learning document

### ISSUE THREAD

**Issue by Peaceuai** [microsoft/Windows-Machine-Learning#531]
I found two problems in the following official website documents

Document address: https://learn.microsoft.com/en-us/windows/ai/windows-ml/tutorials/pytorch-train-model

The first problem is that the code in the following picture example does not call the model.train() method to train the model in each epoch, because the train() method calls the testAccuracy() method and the testAccuracy() method calls the model.eval() method, which will cause the model to be trained through model.eval() in subsequent model training. In this case, if there is a Dropout layer or a BatchNorm layer in the CNN network, this will cause both of them to not work.
![Image](https://github.com/user-attachments/assets/a01a12a4-6b59-4051-a686-74f654976478)

The second problem is that the text in the following picture shows that CNN is written as CCN.
![Image](https://github.com/user-attachments/assets/3387a686-f5ec-45f3-8653-5eb4de1f5c62)

**Comment by 4rnv** [comment]
In the same section the convolutional layers are defined as 1,2,4,5 instead of 1,2,3,4. Minor nitpick, but for a second I thought I had missed something.
